python/scientific-python.ipynb

```diff
@@ -13,18 +13,24 @@
     "\n",
     "Überblick über (fast) alles weitere was SciPy kann gibt es hier:\n",
     "\n",
-    "- https://github.com/MaxNoe/scientific_python_notebooks/blob/master/scipy.ipynb"
+    "- https://github.com/MaxNoe/scientific_python_notebooks/blob/master/scipy.ipynb\n",
+    "\n",
+    "Diese Einführung konzentriert sich auf die folgenden Themen:\n",
+    "\n",
+    "- Abrufen von physikalischen Naturkonstanten samt Unsicherheit und Einheit\n",
+    "- Lineare und Nichtlineare Regression"
```

Comment:
In the python-notebook, we use a table of content.
Use 
```
%%javascript
$.getScript('https://kmahelona.github.io/ipython_notebook_goodies/ipython_notebook_toc.js')
```
in a Code cell and
```
# Inhalt

<div id="toc"></div>
```
in a Markdown cell.